nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/test/java/org/apache/nifi/processors/kafka/TestGetKafka.java

```diff
@@ -45,13 +45,18 @@ public static void configureLogging() {
     }
 
     @Test
-    @Ignore("Intended only for local tests to verify functionality.")
     public void testIntegrationLocally() {
+        EmbeddedKafka kafka = new EmbeddedKafka();
+        kafka.start();
+
+        kafka.addTopic("testX", 1);
+        kafka.sendMessage("testX");
+
         final TestRunner runner = TestRunners.newTestRunner(GetKafka.class);
-        runner.setProperty(GetKafka.ZOOKEEPER_CONNECTION_STRING, "192.168.0.101:2181");
+        runner.setProperty(GetKafka.ZOOKEEPER_CONNECTION_STRING, "localhost:2181");
         runner.setProperty(GetKafka.TOPIC, "testX");
-        runner.setProperty(GetKafka.KAFKA_TIMEOUT, "3 secs");
-        runner.setProperty(GetKafka.ZOOKEEPER_TIMEOUT, "3 secs");
+        // runner.setProperty(GetKafka.KAFKA_TIMEOUT, "3 secs");
```

Comment:
If this line and the next are not needed, can you remove them rather than commenting them out?